change/office-ui-fabric-react-2020-03-17-17-21-14-iconbutton.json

```
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
```

Should be patch since it's not adding a new API.
```suggestion
  "type": "patch",
```